public/app/features/variables/query/queryRunners.test.ts

```diff
@@ -196,7 +196,7 @@ describe('QueryRunners', () => {
 
       it('then it should call runRequest with correct args', () => {
         expect(runRequest).toHaveBeenCalledTimes(1);
-        expect(runRequest).toHaveBeenCalledWith(datasource, {}, datasource.variables.query);
```

Comment:
Not sure about this test change but not sure how meaningful the tests actually are. Seems like they test internal implementation too much and are a bit brittle as this change should not make the tests fail.